src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -262,9 +420,14 @@ trait ModuleValue[T]
 
 object InModuleBody
 {
+  /** Used to inject code snippets to be evaluated in  [[LazyModuleImp.instantiate]] in the current [[LazyModule.scope]].
+    * It can be used to create additional hardware outside of the  [[LazyModule.children]] and
+    * connections other than the internal [[BaseNode]] connections or additional IOs aside from the  [[AutoBundle]] 
+    * */
```

Comment:
```suggestion
  /** Used to inject code snippets to be evaluated in  [[LazyModuleImp.instantiate]]
    *  in the current [[LazyModule.scope]].
    * It can be used to create additional hardware outside of the  [[LazyModule.children]],
    * connections other than the internal [[BaseNode]] connections,
    * or additional IOs aside from the  [[AutoBundle]] 
    * */
```